studio/deskStructure.js

```diff
@@ -49,8 +49,8 @@ export default () =>
               S.documentTypeListItem('imageBesideText').title('Image beside Text'),
               S.documentTypeListItem('sideBySideImages').title('Side by side Images'),
               S.documentTypeListItem('doubleOptions').title('Double Options'),
-              S.documentTypeListItem('imageWithText').title('Image'),
-              S.documentTypeListItem('mailchimp').title('Mailchimp Newsletter Signup'),
+              // S.documentTypeListItem('imageWithText').title('Image'),
```

Comment:
Talvez apagar esse comentário se não for usá-los posteriormente

Reply:
Como poderemos utilizar no futuro, tirei temporariamente para limpar um pouco o que vai ser exibido no menu quando o usuário clicar em Sections. Caso decidamos mais pra frente que não vamos usar de jeito nenhum, aí vamos remover tanto essas linhas quanto os arquivos do CMS e do front-end.